docs/reference/run.md

```diff
@@ -1232,7 +1232,13 @@ The followings examples are all valid:
 ### Working directory
 
 The default working directory for running binaries within a container is the
-root directory (`/`). It is possible to set a different working directory with the
-Dockerfile `WORKDIR` command. The operator can override this with:
+root directory (`/`). The default working directory of an image is set using
+the Dockerfile `WORKDIR` command. You can override the default working
+directory for an image using the `-w` (or `--workdir`) flag for the `docker
+run` command:
 
-    -w="", --workdir="": Working directory inside the container
+```text
+$ docker run -w /home/$USER -it alpine
```

Comment:
We should probably avoid `$USER` here, as that depends on the variable being set on the host. Perhaps some other example that doesn't depend on an env-var?

Perhaps also make the example illustrate the result;


```bash
$ docker run --rm -w /my/workdir alpine pwd
/my/workdir
```